tools/refguide_check.py

```diff
@@ -505,7 +505,7 @@ class Checker(doctest.OutputChecker):
     obj_pattern = re.compile('at 0x[0-9a-fA-F]+>')
     int_pattern = re.compile('^[0-9]+L?$')
     vanilla = doctest.OutputChecker()
-    rndm_markers = {'# random', '# Random', '#random', '#Random', "# may vary"}
+    rndm_markers = {'# random', '# Random', '#random', '#Random', "# may vary", "# uninitialized, unpredictable", "#uninitialized, unpredictable"}
```

Comment:
Line is too long - should be limited to 80 characters.

Reply:
Hi @mattip , thanks! I will update it. Should the lines in the examples also be limited to 80 characters? I am asking because the one in `empty_like` is already 69 characters. If it is limited to 80 characters, I can only put it in a new line.

Reply:
Practicality beats purity. If it is easy to change (like this one) and wayyyyyy over the limit, then the line should be broken. I am not sure if breaking the example output lines works with refguide-check, so maybe not worth the trouble.